GPflow/likelihoods.py

```diff
@@ -436,7 +436,7 @@ def predict_mean_and_var(self, Fmu, Fvar):
             # To compute this, we'll compute the density for each possible output
             possible_outputs = [tf.fill(tf.stack([tf.shape(Fmu)[0], 1]), np.array(i, dtype=np.int64)) for i in
                                 range(self.num_classes)]
-            ps = [self.predict_density(Fmu, Fvar, po) for po in possible_outputs]
+            ps = [tf.exp(self.predict_density(Fmu, Fvar, po)) for po in possible_outputs]
```

Comment:
Not really too relevant as we no longer do this, but I'm not too sure whether exp(lnA) is simplified to A. The other way round should occur I think (see line 325 of https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/xla/service/algebraic_simplifier.cc), but I could be wrong.

 I have found it difficult to ever find any documentation for the simplifications Tensorflow does and would be grateful if you had a reference.